main.py

```
@@ -18,47 +18,37 @@
 
 dotenv.load_dotenv(".env")
 
-# Setup paths
-root_folder = "."
-projectPath = Path(rf'{root_folder}')
-dataPath = projectPath / 'data'
-configPath = projectPath / 'config'
-credsPath = projectPath / 'credentials'
+def main():
```

Ruff (0.8.2)

Missing return type annotation for public function `main`

Add return type annotation: `None`

(ANN201)